src/server/Document/__snapshots__/component.test.jsx.snap

```diff
@@ -5,16 +5,35 @@ exports[`Document Component should render AMP version correctly 1`] = `
   lang="test"
 >
   <head>
-    <title
-      data-react-helmet="true"
-    >
-      Test title
-    </title>
     <meta
       content="width=device-width, initial-scale=1, minimum-scale=1"
       data-react-helmet="true"
       name="viewport"
     />
+    <script
```

Comment:
React 19 changes some behaviour of `<head>` related tags, which likely accounts for this snapshot diff: https://react.dev/blog/2024/12/05/react-19#support-for-async-scripts

Reply:
I think this is OK - it's just changing the order of the content in the snapshot by the looks of it?

Reply:
Yea I'll check and compare the ordering to see if there is any impact.